node/src/service.rs

```diff
@@ -30,24 +30,25 @@ use cumulus_client_service::{
     prepare_node_config, start_collator, start_full_node, StartCollatorParams, StartFullNodeParams,
 };
 use cumulus_primitives_core::{
-    relay_chain::v1::{Hash as PHash, PersistedValidationData},
+    relay_chain::v2::{Hash as PHash, PersistedValidationData},
     ParaId,
 };
 use cumulus_relay_chain_inprocess_interface::build_inprocess_relay_chain;
 use cumulus_relay_chain_interface::{RelayChainError, RelayChainInterface, RelayChainResult};
 use cumulus_relay_chain_rpc_interface::RelayChainRPCInterface;
+use jsonrpsee::RpcModule;
```

Comment:
Have you E2E-tested syncing to dolphin frontend via the new rpc lib with a polkadot-launched dolphin full node?

Reply:
Working without unsafe flag locally. Not sure why

Reply:
This is the result from local nodes.
```
  Node RPC Test
>>>>>>>>> UTXO INSERT START >>>>>>>>
>>>> Inserting 1 big batch UTXOs
starting utxo idx: 0
📝 1 batch utxos insertion finalized.
total wait: 42 sec.
>>>> Complete 1 big batch with 1024 UTXOs
>>>>>>>>> UTXO INSERT DONE >>>>>>>>
>>>> Inserting void numbers: 1024 per batch, 1 batchs
start vn batch 0
📝 1 batchs void number insertion finalized.
total wait: 47 sec.
>>>> Complete inserting 1024 void numbers
    ✔ Check RPC result (89910ms)


  1 passing (1m)

Done in 93.23s.
```

Reply:
There's one more `yarn ts-node rpc_performance.ts`

Reply:
Performance test for manta-pay rpc.
```
>>>>>>>>> UTXO INSERT START >>>>>>>>
>>>> Inserting 1 big batch UTXOs
starting utxo idx: 0
📝 1 batch utxos insertion finalized.
📝 2 batch utxos insertion finalized.
📝 3 batch utxos insertion finalized.
📝 4 batch utxos insertion finalized.
total wait: 78 sec.
>>>> Complete 1 big batch with 16384 UTXOs
>>>>>>>>> UTXO INSERT DONE >>>>>>>>
>>>> Inserting void numbers: 4096 per batch, 2 batchs
start vn batch 0
start vn batch 1
📝 1 batchs void number insertion finalized.
📝 2 batchs void number insertion finalized.
total wait: 47 sec.
>>>> Complete inserting 8192 void numbers
ledger diff receiver size: 16384
ledger diff void number size: 8192
single rpc sync time: 605 ms
Done in 129.43s.
```